.github/workflows/lint.yml

```diff
@@ -10,10 +10,9 @@ jobs:
     # We need to run the linter on the same image we use for building, since it
     # needs the C libraries installed for the dependencies to typecheck.
     runs-on: ubuntu-latest
-    container: tendermintdev/docker-tm-db-testing
+    container: archlinux
```

Comment:
why this change?.

Reply:
the former image only supports ancient rocksdb, and this prevents meaningful comparisons